sdks/python/apache_beam/transforms/core.py

```diff
@@ -2265,10 +2266,18 @@ def __init__(self, windowfn, triggerfn=None, accumulation_mode=None,
       raise ValueError(
           'window fn (%s) does not have a determanistic coder (%s)' % (
               windowfn, windowfn.get_window_coder()))
+    if isinstance(allowed_lateness, Duration):
```

Comment:
I think we can just do `Duration.of(allowed_lateness)`.

Reply:
got it, thanks.